cms/static/sass/views/_group-configuration.scss

```diff
@@ -77,7 +77,14 @@
               display: inline-block;
               width: ($baseline*0.75);
               vertical-align: baseline;
-              margin-left: -$baseline;
+              @include margin-left($baseline);
```

Comment:
I've had the comment earlier. I flipped it not just to RTL the interface but it look better to me to flip the direction.

I can flip it back, I really don't mind.